README.md

```diff
@@ -165,7 +178,7 @@ rightArrowIcon | react node | `<Icon>keyboard_arrow_right</Icon>`| Right arrow i
 dateRangeIcon | react node | `<Icon>date_range</Icon>`| Date tab icon 
 timeIcon | react node | `<Icon>access_time</Icon>`| Time tab icon
 ampm | boolean | true | 12h/24h view for hour selection clock
-shouldDisableDate | (date: Moment) => boolean | () => false | Allow to disable custom date in calendar
+shouldDisableDate | (date: Moment | Date) => boolean | () => false | Allow to disable custom date in calendar
```

Comment:
can we avoid passing moment object to this callback?

Reply:
No. We are poviding moment for api consistent. Moment utils is a way to use moment in this callback.

Reply:
what if we pass `utils` as second argument? This will allow data processing

Reply:
@cherniavskii read my previous answer :)